examples/ExampleApp/app/src/main/kotlin/io/embrace/android/exampleapp/MainActivity.kt

```diff
@@ -7,31 +7,41 @@ import androidx.activity.enableEdgeToEdge
 import androidx.navigation.compose.NavHost
 import androidx.navigation.compose.composable
 import androidx.navigation.compose.rememberNavController
+import io.embrace.android.embracesdk.Embrace
+import io.embrace.android.embracesdk.spans.Workflow
 import io.embrace.android.exampleapp.ui.CodeExample
 import io.embrace.android.exampleapp.ui.CodeExampleDetailScreen
 import io.embrace.android.exampleapp.ui.CodeExampleListScreen
 import io.embrace.android.exampleapp.ui.theme.ExampleAppTheme
 
 class MainActivity : ComponentActivity() {
 
+    private val operation: Workflow = Embrace.getInstance().createOperation("app-screen")
+
     override fun onCreate(savedInstanceState: Bundle?) {
         super.onCreate(savedInstanceState)
+        operation.start("load")
         val examples = CodeExample.entries
         enableEdgeToEdge()
         setContent {
             ExampleAppTheme {
                 val navController = rememberNavController()
                 NavHost(navController = navController, startDestination = "main") {
                     composable("main") {
-                        CodeExampleListScreen(navController, examples)
+                        CodeExampleListScreen(navController, examples, operation)
                     }
                     examples.forEach { example ->
                         composable(example.route) {
-                            CodeExampleDetailScreen(navController, example)
+                            CodeExampleDetailScreen(navController, example, operation)
                         }
                     }
                 }
             }
         }
     }
+
+    override fun onPause() {
+        super.onPause()
+        operation.end()
```

Comment:
What's the behavior if folks attempt to add to an ended span? Presumably the same as whatever the behavior is for existing spans, right?